frontend/src/tests/components/Shift/ShiftForm.test.js

```diff
@@ -0,0 +1,118 @@
+import { fireEvent, render, screen, waitFor } from "@testing-library/react";
+import { BrowserRouter as Router } from "react-router-dom";
+
+import ShiftForm from "main/components/Shift/ShiftForm";
+import shiftFixtures from "fixtures/shiftFixtures";
+
+import { QueryClient, QueryClientProvider } from "react-query";
+
+const mockedNavigate = jest.fn();
+
+jest.mock('react-router-dom', () => ({
+    ...jest.requireActual('react-router-dom'),
+    useNavigate: () => mockedNavigate
+}));
+
+describe("ShiftForm tests", () => {
+    const queryClient = new QueryClient();
+
+    const expectedHeaders = ["Day of Week", "Start Time", "End Time", "Driver ID", "Backup Driver ID"];
+    const testId = "ShiftForm";
+
+    test("renders correctly with no initialContents", async () => {
+        render(
+            <QueryClientProvider client={queryClient}>
+                <Router>
+                    <ShiftForm />
+                </Router>
+            </QueryClientProvider>
+        );
+
+        expect(await screen.findByText(/Create/)).toBeInTheDocument();
+
+        expectedHeaders.forEach((headerText) => {
+            const header = screen.getByText(headerText);
+            expect(header).toBeInTheDocument();
+          });
+        expect(screen.getByTestId(`${testId}-cancel`)).toBeInTheDocument;
+        expect(screen.getByTestId(`${testId}-day`)).toBeInTheDocument;
```

Comment:
And on all of the rest as well.  That will get rid of one set of the eslint failures.

```suggestion
        expect(screen.getByTestId(`${testId}-day`)).toBeInTheDocument();
```